examples/h2o-classification/h2o-example.ipynb

```diff
@@ -0,0 +1,218 @@
+    "prostate[\"CAPSULE\"] = prostate[\"CAPSULE\"].asfactor()\n",
+    "# construct and define the estimator object \n",
+    "model = H2ODeepLearningEstimator(activation = \"Tanh\", hidden = [10, 10, 10], epochs = 100)\n",
+    "# train the model on the whole prostate dataset\n",
+    "model.train(x = list(set(prostate.columns) - set([\"ID\",\"CAPSULE\"])), y =\"CAPSULE\", training_frame = prostate)\n",
+    "model.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "predictions=model.predict(prostate)\n",
+    "predictions.show()\n",
+    "print(type(model))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "path = h2o.save_model(model=model, path=\"./model\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Export with BentoML"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile h2o_model_service.py\n",
+    "import pandas as pd\n",
+    "import h2o\n",
+    "import bentoml\n",
+    "from bentoml.artifact import H2oModelArtifact\n",
+    "from bentoml.handlers import DataframeHandler\n",
+    "\n",
+    "@bentoml.artifacts([H2oModelArtifact('model')])\n",
+    "@bentoml.env(conda_dependencies=[\"h2o\", \"pandas\"])\n",
+    "class H2oModelService(bentoml.BentoService):\n",
+    "\n",
+    "    @bentoml.api(DataframeHandler)\n",
+    "    def predict(self, df):     \n",
+    "        hf = h2o.H2OFrame(df)\n",
+    "        return self.artifacts.model.predict(hf)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from h2o_model_service import H2oModelService\n",
+    "\n",
+    "# Initialize bentoML model with artifacts\n",
+    "\n",
+    "bento_model = H2oModelService.pack(\n",
+    "    model=model\n",
+    ")\n",
+    "\n",
+    "# Save bentoML model to directory\n",
+    "saved_path = bento_model.save(\"/tmp/bento\")\n",
+    "\n",
+    "# print the directory containing exported model archive (prefixed with model name and version)\n",
+    "print(saved_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import bentoml\n",
+    "import pandas as pd\n",
+    "\n",
+    "df = pd.read_csv(\"https://raw.githubusercontent.com/multicode/h2o-notebook/master/prostate.csv\")\n",
+    "# Load exported bentoML model archive from path\n",
+    "bento_model = bentoml.load(saved_path)\n",
+    "bento_model.predict(df)"
```

Comment:
could you also add 'curl' and 'cli' example?

Reply:
done